src/pages/Favorites.jsx

```diff
@@ -147,43 +143,48 @@ export default function Favorites() {
     setFavourites(RESAPI.data.products);
   }, [USERMOCK]);
 
-  if (!USERMOCK) {
-    return <Redirect to="/" />;
-  } else {
-    return (
-      <div className="favsPage">
-        <div>
-          <div className="favsPage_welcome">
-            Welcome to your favourites {USERMOCK} 😎!
+  if (user) {
+    if (!USERMOCK) {
```

Comment:
Por qué seguimos usando USERMOCK aquí, si en la línea anterior ya se está comprobando si hay user o no. Creo que esa línea no debería estar. Si tiene que estar, me lo explicáis luego, vale?